### PR TITLE
Qt 5.15 deprecation

### PR DESCRIPTION
**Pull request by zomfg**

### CODE REVIEW

File: Software/src/LogWriter.cpp [hunk 1]
```diff
@@ -29,33 +29,42 @@ int LogWriter::initWith(const QString& logsDirPath)
...
 
 		const QDateTime currentDateTime(QDateTime::currentDateTime());
 		m_logStream << currentDateTime.date().toString("yyyy_MM_dd") << " ";
-		m_logStream << currentDateTime.time().toString("hh:mm:ss:zzz") << " Prismatik " << VERSION_STR << endl;
+		m_logStream << currentDateTime.time().toString("hh:mm:ss:zzz") << " Prismatik " << VERSION_STR;
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+		m_logStream << Qt::endl;
+#else
+		m_logStream << endl;
+#endif
```
Comment (psieg) [inline Software/src/LogWriter.cpp:61]: Urgh.

Comment (psieg) [reply Software/src/LogWriter.cpp:61]: Not blaming anyone, but that's messy

**Comment by psieg** [comment]
I guess I'll wait with the release until you're done updating things ;)

File: Software/src/LogWriter.cpp (hunk 1 above)
Comment (zomfg) [reply Software/src/LogWriter.cpp:61]: For sure.. well when 5.12 is gonna be dropped.. in 2022.. it'll be better :p

**Comment by zomfg** [comment]
xcode was the last thing I had in mind 👍

**Pull request merged**